### ISSUE TITLE
A full-fleged installer so that WineVDM could be included in RosAPPs directory?

### ISSUE THREAD

**Issue by Anixx** [otya128/winevdm#355]
Since WineVDM now works on Reactos, it would be great to have it in its software catalog.

**Comment by cracyc** [comment]
https://github.com/otya128/winevdm/issues/244 maybe?

**Comment by Anixx** [comment]
No, I meant a probably .msi installer, compatible with RosAPPs catalog.

**Comment by learn-more** [comment]
Which version of WineVDM did you test on which version of ReactOS?

**Comment by otya128** [comment]
I have tested that [this build](https://ci.appveyor.com/project/otya128/winevdm/builds/26773988/job/c5yxs6ucmbe0wvft) works on ReactOS 0.4.11.

**Comment by jeditobe** [comment]
Hi ^)  Any news on this?

**Comment by Zero3K** [comment]
Please check https://github.com/reactos/rapps-db/pull/159 for a link to an installer.
